src/features/MentorPage/components/MentorsFilter/Skills/BottomBar/PageSizeDropdown/const.ts

```diff
@@ -0,0 +1,2 @@
+export const defaultPageSize = 25;
```

Comment:
Hmm could the file be named constants.ts etc. There's something weird with `const` as the filename that I can't put my finger on 😆